### DISCUSSION TITLE
Disable v86 auto save sessions

### DISCUSSION THREAD

**Discussion by rocheston** [DustinBrett/daedalOS#323] [Feature Requests]
Dustin, is there any way to disable the feature of saving and loading v86 sessions? This seems to slow down Alpine and Bodhi Linux, which are big in size.

I would like to know how to disable the auto save/load state feature in v86 completely.

Thank you.

**Comment by DustinBrett** [comment]
I haven't considered this option for the main app as in general I like the auto save snapshot. But if you want to modify the code, `closeDiskImage` only needs to have:

```js
try {
  emulator[diskImageUrl]?.destroy();
} catch {
  // Ignore failures on destroy
}
```

Everything else could be removed to stop auto saving snapshots.

https://github.com/DustinBrett/daedalOS/blob/main/components/apps/V86/useV86.ts#L66

**Comment by rocheston** [comment]
I attempted the suggested approach, but it resulted in numerous errors during the build process. Here's the relevant code snippet:


const closeDiskImage = useCallback(
  async (diskImageUrl: string): Promise<void> => {
    try {
      emulator[diskImageUrl]?.destroy();
    } catch {
      // Ignore failures on destroy
    }
  },
  [emulator]
);


Could you please share the complete code? I appreciate your assistance in advance.

**Comment by rocheston** [comment]
Can you provide me with this file https://github.com/DustinBrett/daedalOS/blob/main/components/apps/V86/useV86.ts 
with updated code. I'm having trouble compiling with the above changes. Thanks again Dustin.

**Comment by DustinBrett** [comment]
I can offer suggestions, but I hadn't planned to start sending out modified code on request. If you are going to be working with the code and getting build errors, that is normal and you will need to do a bit of coding along the way. I'd suggest you start from a working build and then comment out code blocks related to the file system.
